lambda/README.md

```diff
@@ -1,15 +1,15 @@
 # Replacement Glue Crawler Lambda
 
-One of our primary uses of glutil is as a Glue Crawler analog for table and data that the Glue Crawler just isn't great at crawling.
+One of our primary uses of `glutil` is as a Glue Crawler analog for tables and data that the Glue Crawler cannot crawl.
```

Comment:
Same comment about redundancy here.

Reply:
I'm fine with restating, as someone could be linked directly to the lambda docs and not bounce up to the main README. Not saying its likely but 🤷‍♂

Reply:
Agree with Phil, because it's a separate document. I don't think redundancy is a bad thing in documentation because of how people use documentation. People mostly read the sections they need and not the entire document/document set.